lib/src/st2110/st_rx_video_session.c

```diff
@@ -2774,7 +2776,11 @@ static int rv_pkt_rx_tasklet(struct st_rx_video_session_impl* s) {
         s->in_continuous_burst[s_port] = true;
       }
 
-      rv_handle_mbuf(&s->priv[s_port], &mbuf[0], rv);
+      check_err_code = rv_handle_mbuf(&s->priv[s_port], &mbuf[0], rv);
+      if (check_err_code < 0) {
+        err("%s(%d,%d), handle mbuf fail %d\n", __func__, s->idx, s_port, check_err_code);
+      }
+
```

Comment:
This is critical path 
if we run into that we will have quite a lot of error messages, 
i think its fine the way it is but mby better would be to add them to statistic threads? 
in critical path we may handle few hundred of those rapidly, which may be more information that it's worth